kitchen.yml

```diff
@@ -69,6 +69,12 @@ suites:
         - test/integration/inputs-file
       input_files:
         - test/integration/profile-inputs.yml
+  - name: with_waivers
+    verifier:
+      inspec_tests:
+        - test/integration/with_waivers
+      waiver_files:
+        - test/integration/with_waivers.yml
```

Comment:
Are we referring to `with_waivers.yml` file in `test/integration` directory? I am unable to find any such file.